File: java/LegislatureSection.java
// Author : Mustafa Mulla
// Roll No : 27
// Title : Goa legistalive assembly
// Start date : 15/07/2024
// Modified date :  22/07/2024
// Description : Goa Legislative Assembly's various sections
import java.util.ArrayList;
import java.util.List;

public class LegislatureSection extends Goa_legislative{
    private List<Subsection> subsections;

    public LegislatureSection(){
        super("Legislature");
        this.subsections= new ArrayList<>();

        subsections.add(new Subsection("Speaker : Shri. Ramesh Tawadkar"));
        subsections.add(new Subsection("Deputy Speaker : Shri. Joshua De Souza"));
        // subsections.add(new Subsection("MLA's"));
        subsections.add(new Subsection("Secretary : Smt. Namrata Ulman"));
        // subsections.add(new Subsection("Infrastructure"));
    }

    @Override
    public List<Subsection> getSubsections(){
        return subsections;
    }
}

File: python/legislature_section.py
# Author : Mustafa Mulla
# Roll No : 27
from typing import List
from goa_legislative import GoaLegislative
from subsection import Subsection

class LegislatureSection(GoaLegislative):
    def __init__(self):
        super().__init__("Legislature")
        self._subsections = [
            Subsection("Speaker : Shri. Ramesh Tawadkar"),
            Subsection("Deputy Speaker : Shri. Joshua De Souza"),
            Subsection("Secretary : Smt. Namrata Ulman")
        ]

    def get_subsections(self) -> List[Subsection]:
        return self._subsections
